### DISCUSSION TITLE
Create a MDAKits forum?

### DISCUSSION THREAD

**Discussion by orbeckst** [MDAnalysis/mdanalysis#4411] [General]
I think it could be useful to have a group/forum/category for [MDAKits](https://mdakits.mdanalysis.org/). For a start, we would then have a dedicated place where we could send anyone who has problems with creating or maintaining a kit, which is a bit different from standard user/developer discussions.

Alternatively, we could use the [Development forum](https://github.com/MDAnalysis/mdanalysis/discussions/categories/development).

Do you have opinions one way or another, especially @RMeli and @fiona-naughton @lilyminium @IAlibay ?

**Comment by orbeckst** [comment]
In PR https://github.com/MDAnalysis/MDAKits/pull/112 I am currently linking to the Discussions in total but if we create a MDAKits forum then I would link to it specifically.

**Reply by IAlibay** [reply]
I guess it could make a decent place to store Q&A style answers for issues? I guess it depends on how we want to make these notes going forward.

**Reply by orbeckst** [reply]
I wasn't so much thinking as storage (I think in our meeting we didn't want to use Discussions for that) but rather as a discussion place where the team can become aware of problems. Once there's a solution found in the discussions I would clean it up and immortalize it in the curated Troubleshooting page.

**Comment by RMeli** [comment]
I think a MDAKit category or even a section (if we have popular MDAKits requiring their own tag) is a great idea!

One possibility would be to open GitHub Discussions on the relevant repo (MDAKit?), but maybe it becomes too dispersive and would be better to have everything in one place here.

**Reply by hmacdope** [reply]
I would second a single place. :)

**Reply by IAlibay** [reply]
+1 single place - especially if you can transfer issues to the discussion board of another repo

**Reply by orbeckst** [reply]
+1 single place

**Reply by RMeli** [reply]
Sounds good. I created a section, we just need to populate it with categories.

**Reply by orbeckst** [reply]
I would keep it simple and not do too many categories. Maybe
- MDAKits Q&A
- MDAKits discussions

**Reply by RMeli** [reply]
Added the MDAKit section and the requested categories.

**Comment by orbeckst** [comment]
@lilyminium @fiona-naughton did you have an opinion on creating a MDAKits section in the forum?

**Comment by lilyminium** [comment]
Sounds good, definitely prefer a single place over multiple Discussions!
